tests/testthat/test-24-1-loadReg.R

```diff
@@ -1,3 +1,4 @@
+context("loadReg() is defined within rloadest. The functions required to implement loadModelInterface are defined in loadReg.R within the current package.")
```

Comment:
a useful note, though `context()` might not ultimately be the best place for this because the testthat output is more readable when the context is 1-2 words. Let's leave it for now as a reminder to put that info somewhere else prominent soon.